slurm-cluster-chart/values.yaml

```
@@ -8,22 +8,27 @@ nfs:
   mountPath: /home
   claimName: rook-nfs-pv-claim
 
-sqlImage: mariadb:10.10
-
-databaseStorage: 100Mi
-
+# Values for Slurm's database container
+mySQL:
```

```suggestion
database:
```
MariaDB != MySQL so I'd prefer this heading to be generic. Will also need to change this anywhere in the templates that this value is referenced.